web/angular-wallet/src/app/main/send-burst/send-burst-form/send-burst-form.component.ts

```diff
@@ -101,21 +101,16 @@ export class SendBurstFormComponent extends UnsubscribeOnDestroy implements OnIn
 
     try {
       this.isSending = true;
-      await this.transactionService.sendMoney({
-        transaction: {
-          // FIX: amountNQT is actually in burst
-          amountNQT: this.amount,
-          feeNQT: this.fee,
-          attachment: this.getMessage(),
-          deadline: parseFloat(this.deadline) * 60,
-          fullHash: this.fullHash,
-          type: 1
-        },
-        pin: this.pin,
+      await this.transactionService.sendBurst({
+        amount: convertNumberToNQTString(parseFloat(this.amount)),
+        fee: convertNumberToNQTString(parseFloat(this.fee)),
+        recipientId: convertAddressToNumericId(this.recipient.addressRS),
         keys: this.account.keys,
-        recipientAddress: addressRS,
+        pin: this.pin,
+        message: this.message,
+        shouldEncryptMessage: this.encrypt,
+        deadline: 1440
```

Comment:
not really considered yet - I ask myself, if we need that. Still don't understand the impact of a sent deadline